src/UpgradeRule/PHP/ClassToTraitRule.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SilverStripe\Upgrader\UpgradeRule\PHP;
+
+use SilverStripe\Upgrader\CodeCollection\CodeChangeSet;
+use SilverStripe\Upgrader\CodeCollection\ItemInterface;
+use SilverStripe\Upgrader\UpgradeRule\PHP\Visitor\ClassToTraitVisitor;
+use SilverStripe\Upgrader\UpgradeRule\PHP\Visitor\ParentConnector;
+use SilverStripe\Upgrader\Util\MutableSource;
+
+/**
+ * Replaces occurrences of class extensions with traits.
+ *
+ * Class ClassToTraitRule
+ * @package SilverStripe\Upgrader\UpgradeRule\PHP
```

Comment:
Let's avoid using `@package`

Reply:
This whole (autogenerated from PHPStorm) doc block is pointless, I'd suggest disabling it in your IDE templates